script/package-debian.ts

```diff
@@ -16,7 +16,7 @@ type DebianOptions = {
   // required
   src: string
   dest: string
-  arch: 'amd64' | 'i386' | 'arm64'
+  arch: 'amd64' | 'i386' | 'arm64' | 'armhf'
```

Comment:
:art: As a first step for this work I'd happily take the API changes required to enable ARM support. I think the changes below this line will impact CI, but seeing this helps me think about how we could make this smoother (detect the environment and then pass this through to the packaging stages).

If you wanted to open a fresh PR with just those API changes let me know, otherwise I'll see if I can make time over the weekend to do the same...

Reply:
I force pushed removed those changes. making the other api changes isn't too urgent (since dugite-native still needs changes to its buildscripts as well). but this is a start (and means less patching when I go to build)